doc/source/user_guide/io.rst

```diff
@@ -5226,6 +5226,7 @@ See the `Full Documentation <https://github.com/wesm/feather>`__.
 Write to a feather file.
 
 .. ipython:: python
+   :okwarning:
```

Comment:
What warning are these examples showing?

Reply:
i dont remember off the top of my head, just that it was one of the is_foo checks

Reply:
Does it mean pyarrow are using these methods then?

Reply:
Looks like pyarrow/pandas_compat.py:373 is using is_sparse

Reply:
Okay I think it's reasonable for them to use the isinstance check

Reply:
gentle ping